lib/src/lds/mrz.dart

```diff
@@ -31,40 +32,73 @@ class MRZ {
   late String _optData;
   String? _optData2;
 
-  MRZ(Uint8List encodedMRZ) {
+  final Uint8List _encoded;
+
+  MRZ(Uint8List encodedMRZ) : _encoded = encodedMRZ {
     _parse(encodedMRZ);
   }
 
-  static int calculateCheckDigit(String checkString)  {
+  Uint8List toBytes() {
+    return _encoded;
+  }
+
+  String toEncodedString() {
+    var data = toBytes();
+    final inputStream = InputStream(data);
+    var result = _read(inputStream, data.length);
+
+    return result;
+  }
+
+  static int calculateCheckDigit(String checkString) {
     const charMap = {
-      "0" :  "0",  "1" :  "1",
-      "2" :  "2",  "3" :  "3",
-      "4" :  "4",  "5" :  "5",
-      "6" :  "6",  "7" :  "7",
-      "8" :  "8",  "9" :  "9",
-      "<" :  "0",  " " :  "0",
-      "A" : "10",  "B" : "11",
-      "C" : "12",  "D" : "13",
-      "E" : "14",  "F" : "15",
-      "G" : "16",  "H" : "17",
-      "I" : "18",  "J" : "19",
-      "K" : "20",  "L" : "21",
-      "M" : "22",  "N" : "23",
-      "O" : "24",  "P" : "25",
-      "Q" : "26",  "R" : "27",
-      "S" : "28",  "T" : "29",
-      "U" : "30",  "V" : "31",
-      "W" : "32",  "X" : "33",
-      "Y" : "34",  "Z" : "35"
+      "0": "0",
+      "1": "1",
+      "2": "2",
+      "3": "3",
+      "4": "4",
+      "5": "5",
+      "6": "6",
+      "7": "7",
+      "8": "8",
+      "9": "9",
+      "<": "0",
+      " ": "0",
+      "A": "10",
+      "B": "11",
+      "C": "12",
+      "D": "13",
+      "E": "14",
+      "F": "15",
+      "G": "16",
+      "H": "17",
+      "I": "18",
+      "J": "19",
+      "K": "20",
+      "L": "21",
+      "M": "22",
+      "N": "23",
+      "O": "24",
+      "P": "25",
+      "Q": "26",
+      "R": "27",
+      "S": "28",
+      "T": "29",
+      "U": "30",
+      "V": "31",
+      "W": "32",
+      "X": "33",
+      "Y": "34",
+      "Z": "35"
     };
 
     var sum = 0;
-    var m   = 0;
+    var m = 0;
     const multipliers = [7, 3, 1];
     for (int i = 0; i < checkString.length; i++) {
       final lookup = charMap[checkString[i]];
       final number = int.tryParse(lookup ?? "");
-      if(number == null) {
+      if (number == null) {
```

Comment:
Final nit, revert code format back to original for the whole file (so that all assignments are aligned).